Sources/Client/Model/Channel/Channel.swift

```diff
@@ -164,6 +163,29 @@ public final class Channel: Codable {
         lastMessageDate = try container.decodeIfPresent(Date.self, forKey: .lastMessageDate)
         frozen = try container.decode(Bool.self, forKey: .frozen)
         didLoad = true
+        extraData = decodeChannelExtraData(from: decoder)
+    }
+    
+    /// Safely decode channel extra data and if it fail try to decode only default properties: name, imageURL.
+    private func decodeChannelExtraData(from decoder: Decoder) -> ChannelExtraDataCodable? {
+        do {
+            var extraData = try Self.extraDataType.init(from: decoder) // swiftlint:disable:this explicit_init
+            extraData.imageURL = extraData.imageURL?.removingRandomSVG()
+            return extraData
+            
+        } catch {
+            ClientLogger.log("🐴❌", "Channel extra data decoding error: \(error)")
```

Comment:
It'd be nice to include a log about how we're recovering from this error

Reply:
How it should look like?
```swift
ClientLogger.log("🐴", "Channel extra data recovered as a default ChannelExtraData")
```

Reply:
Maybe instead of
```swift
ClientLogger.log("🐴❌", "Channel extra data decoding error: \(error)")
```
we can:
```swift
ClientLogger.log("🐴❌", "Channel extra data decoding error: \(error). Trying to recover by only decoding name and imageURL")
```

Reply:
@b-onc ^^